components/json-pointer.js

```diff
@@ -0,0 +1,50 @@
+// json-pointer.js
+
+var _ = require('underscore');
+var jsonpointer = require('jsonpointer');
+
+var promiseOutput = require('../src/promise-output');
+var componentFactory = require('../src/noflo-component-factory');
+
+/**
+ * Takes the value of the get pointer for some input and sets that value on the
+ * input at the set pointer. If there is not set pointer provided, this will
+ * return the get pointer value, otherwise it will return the input object.
```

Comment:
Philosophical comment: I would rather avoid components that are so fine grained.  This feels like it should be done directly in JS rather than in an explicit node in a pipeline.  I think I see why you are using it, so let's go ahead with it.  But I hope we can figure out a way in the future to avoid cluttering the pipeline with trivial functions like this.  Grouping things into subgraphs would certainly be one way, but there may be others.  The RDF Pipeline Framework was designed for larger scale (coarser) granularity, in contrast with noflo.